src/config.ts

```diff
@@ -79,17 +83,37 @@ const config = Convict<IConfigSchema>({
         },
     },
     server: {
-        host: {
-            doc: 'Server host',
+        sdk_server_host: {
```

Comment:
I think, we need to have consistency in naming, and we don't need to mix _camelCase_ with _snake_case_ and _UPPER_CASE_